HLshell/mycmd_1.py

```diff
@@ -138,13 +139,12 @@ def run_command(self,func, line):
             func(line)
         except:
             print_err('Error: ' + func.__name__ + '('+line+')')
-
         sys.stdout = sys.__stdout__
         sys.stderr = sys.__stderr__
         out = temp_out.getvalue()
         err = temp_err.getvalue()
         if (out): print(out, end='')
-        if (err): print(err, file=sys.stderr, end='')
+        if (err): print_err(err)
```

Comment:
After the test, the way what I write can perform the same result in both server and client.

Reply:
sure, you can change it back